.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: "yarn"
```

Comment:
I'm not too familiar with yarm, but I think that a similar behavior to `npm ci` would be something like `yarn install --immutable`

Reply:
I checked the documentation (https://github.com/actions/setup-node#caching-global-packages-data) that indicates it is the package manager and not a command.